.github/workflows/ci_windows_x64_msvc.yml

```diff
@@ -21,6 +21,7 @@ concurrency:
 
 jobs:
   windows_x64_msvc:
+    if: ${{ github.repository_owner == 'iree-org' || github.event_name != 'schedule' }}
```

Comment:
Here we could ideally also select the `runs-on` based on the repository[_owner], if this comment were addressed: https://github.com/iree-org/iree/pull/18967#discussion_r1841373180

> Ideally we could make this workflow more agnostic to the specific runner, so it can run in forks. One code pattern for that is https://github.com/openxla/stablehlo/blob/b1c1115f070493845ad92bd4a4461f77a2aa628d/.github/workflows/buildAndTestBazel.yml#L39
> 
> ```yaml
>     runs-on: ${{ github.repository == 'openxla/stablehlo' && 'ubuntu-22.04-64core' ||  'ubuntu-22.04'  }}
> ```
> 
> Something to aim for later. All the build directory setup that is specific to these Azure runners will make that difficult and will limit workflow debugging to only developers with write access to this project.
> 
> A middle ground is possible where the `runs-on` field has a branch _and_ the `Create build dir` step only runs `if ${{ github.repository == ` or `if ${{ runs-on == arc-runner-set`.

Reply:
I think in particular the Windows build wont run on a GH-hosted runner (due to all the sccache and Azure stuff).